Client/Frontend/Onboarding/ViewControllers/IntroViewController.swift

```diff
@@ -225,6 +231,24 @@ extension IntroViewController: OnboardingCardDelegate {
         self.present(controller, animated: true)
     }
 
+    private func presentPrivacyPolicy(url: URL,
+                                      referringPage: ReferringPage = .onboarding) {
+        let privacyPolicyVC = PrivacyPolicyViewController(url: url)
+        let controller: DismissableNavigationViewController
+        let buttonItem = UIBarButtonItem(title: .SettingsSearchDoneButton,
+                                         style: .plain,
+                                         target: self,
+                                         action: #selector(dismissPrivacyPolicyViewController))
+        let theme = BuiltinThemeName(rawValue: LegacyThemeManager.instance.current.name) ?? .normal
+        buttonItem.tintColor = theme == .dark ? UIColor.legacyTheme.homePanel.activityStreamHeaderButton : UIColor.Photon.Blue50
```

Comment:
We should be using the new theme stuff here.

Reply:
For reference https://github.com/mozilla-mobile/firefox-ios/wiki/Theming-system 😄